### PR TITLE
support device message attr error return

### PR DESCRIPTION
**Pull request by orangegzx**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/device.go [hunk 1]
```diff
@@ -159,6 +160,9 @@ func UpdateDeviceAttr(context *dtcontext.DTContext, deviceID string, attributes
 		return nil, nil
 	}
 	dealAttrResult := DealMsgAttr(context, Device.ID, attributes, dealType)
+	if dealAttrResult.Err != nil {
+		return nil, err
+	}
```
Comment (gy95) [inline edge/pkg/devicetwin/dtmanager/device.go:165]: seems reasonable, if err occur, Add, Delete, Update field will be nil.

Comment (gy95) [reply edge/pkg/devicetwin/dtmanager/device.go:165]: better `return nil, nil`
the original logic is always returned `nil, nil`. 😄

Comment (orangegzx) [reply edge/pkg/devicetwin/dtmanager/device.go]: already edited

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**